libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -127,12 +127,20 @@ string IRGenerator::generate(Block const& _block)
 	return generator.code();
 }
 
+string IRGenerator::generate(VariableDeclaration const& _varDecl)
```

Comment:
I think the function name should be more specific.

Reply:
Done